persistence/src/main/java/org/mskcc/cmo/metadb/persistence/neo4j/MetadbSampleRepository.java

```diff
@@ -20,11 +20,10 @@
     @Query("MATCH (sm: Sample {metaDbSampleId: $metaDbSampleId}) "
             + "RETURN sm")
```

Comment:
these queries I just want to double check that the graph is returning what's expected

Reply:
divya: research sample queries were tested, clinical queries have not yet